binr/r2r/load.c

```diff
@@ -25,6 +25,12 @@ static char *readline(char *buf, size_t *linesz) {
 	if (end) {
 		*end = '\0';
 		*linesz = end - buf;
+#if __WINDOWS__
```

Comment:
i dont think this should be a windows-specific feature, unix systems can also encounter this kind of chars in test files if any wuser commits a crlf testfile